src/lib/redux-category.js

```diff
@@ -0,0 +1,12 @@
+export default store => next => (action) => {
+  try {
+    // console.log('__ACTION__', action);
```

Comment:
keep console logs, don't comment out, this is a 'feature' for this lab to track your action and current state